packages/gatsby-cli/src/reporter/index.js

```diff
@@ -65,8 +55,8 @@ module.exports = Object.assign(reporter, {
       error = message
       message = error.message
     }
-    base.error.call(this, message)
-    if (error) console.log(errorFormatter.render(error))
+    // base.error.call(this, message)
```

Comment:
I don't really know what this was doing--seems like just calling reporter.error but from _this_ context.

Probably safe to remove presuming things seem like they're backwards compatible.

Reply:
I actually fixed it immediately when we I was looking at the diff myself